refactor: typo (#535)
oceanroleplay/discord.ts: docs/docs/decorators/commands/slash-group.md

## Example

> In the following example, slash permission commands are grouped together also sub grouped by role and user.

Author: ```suggestion
The following example groups slash permission commands under permissions and subgroups them by role and user.
```
